### DISCUSSION TITLE
Is there a way to switch if someone is using upscroll/downscroll?

### DISCUSSION THREAD

**Discussion by mrflipflops** [KadeArchive/Kade-Engine#1172] [Q&A]
My mod team and I wanted to make a song where in it, the game reads if you're using downscroll with the downscroll variable and switches it. Is there a way to do this or do I have to manipulate the GUI?

**Comment by isakube** [comment]
well i mean if you're using modcharts for that i don't think so
but you can likely switch FlxG.save.data.downScroll (or smth) to false at a certain beat by using `override function curBeat()` in PlayState.hx and then redraw the notes (generateStaticArrows(<0 / 1>))
